src/actionlib/action_client.py

```diff
@@ -495,6 +517,13 @@ def update_feedbacks(self, action_feedback):
         for status in self._get_live_statuses():
             status.update_feedback(action_feedback)
 
+    def set_lost_goal_timestamp_delta(self, lost_goal_timestamp_delta):
```

Comment:
Also consider removing this setter and getter.  Generally you start with direct assignment and if you need additional behaviour, you implement an `@setter` and `@property` methods.